app_feup/lib/controller/networking/network_router.dart

```diff
@@ -225,6 +226,78 @@ class NetworkRouter {
     return tripList;
   }
 
+  // Gets the next arrivals for a given bus stop
+  // Uses FlutterWebviewPlugin in order to scrap the information after the
+  // Javascript is loaded
+  static Future<List<Trip>> getNexArrivalsAfterLoadingJS(
+      String stopCode, BusStopData stopData) async {
+    final flutterWebviewPlugin = FlutterWebviewPlugin();
```

Comment:
Launching this plugin everytime we make the call is rly the best call? can't reuse a webview at all times? sounds less costly. the refactor to its own file would definitely help with this.